README.md

```diff
@@ -214,6 +214,7 @@
 - [nvim-mini/mini.nvim#mini.completion](https://github.com/nvim-mini/mini.nvim/blob/main/readmes/mini-completion.md) - Module of `mini.nvim` for asynchronous two-stage completion. Supports showing completion item info and independent function signature.
 - [deathbeam/autocomplete.nvim](https://github.com/deathbeam/autocomplete.nvim) - Very simple and minimal autocompletion for cmdline and buffer using LSP and Tree-sitter with signature help.
 - [Saghen/blink.cmp](https://github.com/Saghen/blink.cmp) - Really fast completion with LSP & snippet support, along with signature help, cmdline completion, and autobracket support (based on semantic tokens).
+  - [Saghen/blink.compat](https://github.com/Saghen/blink.compat) - Compatibility layer for using `nvim-cmp` sources on `blink.cmp`.
```

Comment:
Extra spaces at the beginning of the row, GitHub might interpret them as a nested list.

Reply:
That might be the intention because `blink.compat` is kind of an add-on for `blink.cmp`

Reply:
@saghen It was indeed. Explained [here](https://github.com/rockerBOO/awesome-neovim/pull/1773#discussion_r2474138837). Don't know what does @Penaz91 think of it.